types/google-map-react/index.d.ts

```diff
@@ -66,30 +66,191 @@ declare namespace googleMapReact {
+            Qz: number;
+        };
+        ControlPosition: {
+            TOP_LEFT: number;
+            TOP_CENTER: number;
+            TOP: number;
+            TOP_RIGHT: number;
+            LEFT_CENTER: number;
+            LEFT_TOP: number;
+            LEFT: number;
+            LEFT_BOTTOM: number;
+            RIGHT_TOP: number;
+            RIGHT: number;
+            RIGHT_CENTER: number;
+            RIGHT_BOTTOM: number;
+            BOTTOM_LEFT: number;
+            BOTTOM_CENTER: number;
+            BOTTOM: number;
+            BOTTOM_RIGHT: number;
+            CENTER: number;
+        };
+        DirectionsStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            MAX_WAYPOINTS_EXCEEDED: string;
+            NOT_FOUND: string;
+        };
+        DirectionsTravelMode: {
+            DRIVING: string;
+            WALKING: string;
+            BICYCLING: string;
+            TRANSIT: string;
+            TWO_WHEELER: string;
+        };
+        DirectionsUnitSystem: {
+            METRIC: number;
+            IMPERIAL: number;
+        };
+        DistanceMatrixStatus: {
+            OK: string;
+            INVALID_REQUEST: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            UNKNOWN_ERROR: string;
+            MAX_ELEMENTS_EXCEEDED: string;
+            MAX_DIMENSIONS_EXCEEDED: string;
+        };
+        DistanceMatrixElementStatus: {
+            OK: string;
+            NOT_FOUND: string;
+            ZERO_RESULTS: string;
+        };
+        ElevationStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            yz: string;
+        };
+        GeocoderLocationType: {
+            ROOFTOP: string;
+            RANGE_INTERPOLATED: string;
+            GEOMETRIC_CENTER: string;
+            APPROXIMATE: string;
+        };
+        GeocoderStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            ERROR: string;
+        };
+        KmlLayerStatus: {
+            UNKNOWN: string;
+            OK: string;
+            INVALID_REQUEST: string;
+            DOCUMENT_NOT_FOUND: string;
+            FETCH_ERROR: string;
+            INVALID_DOCUMENT: string;
+            DOCUMENT_TOO_LARGE: string;
+            LIMITS_EXCEEDED: string;
+            TIMED_OUT: string;
+        };
+        MapTypeControlStyle: {
+            DEFAULT: number;
+            HORIZONTAL_BAR: number;
+            DROPDOWN_MENU: number;
+            INSET: number;
+            INSET_LARGE: number;
+        };
```

Comment:
Could not find `INSET` and `INSET_LARGE` in [the documentation](https://developers.google.com/maps/documentation/javascript/reference/control#MapTypeControlStyle). Where do they come from ?

Reply:
Please see [main comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60329#issuecomment-1126798899) for response.